packages/firebase_storage/example/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ dependencies:
     sdk: flutter
   firebase_storage:
     path: ../
-  firebase_core: ^0.4.0
```

Comment:
This will probably fail until the updated firebase_core is published.

Reply:
0.4.4+2 is already published. Should be fine.